docs/reference/scripts.md

```diff
@@ -200,6 +232,16 @@ function request(ctx, params)
 end
 ```
 
+## set request header from params
+```lua
+function request(ctx, params)
+	ctx.request.header[params[1]] = params[2]
+	if params[1]:lower() == "host" then
+		ctx.request.outgoing_host = params[2]
+	end
+end
+```
```

Comment:
you can also use `preserveHost("false")` in this case, but for doc is nice to see that you can modify outgoing_host!

Reply:
It is basically 1-1 clone of `setRequestHeader("header-name", "header-value")` filter